web/modules/custom/joinup_communities/tallinn/src/Plugin/Field/FieldFormatter/TallinnEntryFormatter.php

```diff
@@ -57,10 +102,17 @@ public function viewElements(FieldItemListInterface $items, $langcode) {
     }
 
     if (!empty($value['uri'])) {
+      $link_title = $value['uri'];
+
+      // Trim the link text to the desired length.
+      if (!empty($settings['trim_length'])) {
+        $link_title = Unicode::truncate($value['uri'], $settings['trim_length'], FALSE, TRUE);
+      }
+
       $element['#uri'] = [
```

Comment:
UX, Accessibility: Now that the URL will be trimmed, the user could not see the whole URL if for some reasons he's interested. Let's add the `title` attribute where we se the full URL so that when he hovers over the link he can read the full address.